src/saving/SaveHelper.cs

```diff
@@ -22,9 +22,58 @@ public enum SaveOrder
         FileSystem,
     }
 
+    public static void RemoveExcessAutoSaves()
+    {
+        int maxAutoSaves = Settings.Instance.MaxAutoSaves;
+        int autoSaveCount = 0;
+
+        List<string> autoSaveNames = new List<string>();
+        List<string> allSaveNames = CreateListOfSaves();
+
+        allSaveNames.Reverse();
+
+        foreach (var save in allSaveNames)
```

Comment:
I think the reverse could be directly in the foreach, LINQ probably has a reverse operation.